package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "prebuild:dev": "npm run clean:build",
     "build:dev": "rollup -c -w",
-    "prebuild:prod": "npm run clean:build",
+    "prebuild:prod": "echo ANALYTICS_ID=${ANALYTICS_ID} >> .env; echo ANSWER=${ANSWER} >> .env; echo API_HOST=${API_HOST} >> .env; echo THE_ONE=${THE_ONE} >> .env; npm run clean:build",
```

Comment:
Is this applicable here in this template?

Reply:
yes, those are the variables for the .env example. You can see they are working [here](https://65f430fc.the-way-svelte-cwa.pages.dev/)

Reply:
Cool. I didn't realize this template actually deployed.

Reply:
now I just need to be able to switch env variables based on branch.